### DISCUSSION TITLE
【提问】有人来过六分街这个地方吗？

### DISCUSSION THREAD

**Discussion by NPC12138590** [share121/inter-knot#720] [其他]
出于工作原因来到六分街，总感觉哪里怪怪的，尤其有时候在玩具店唱片店里，总感觉店长和其他人聊得和我理解的不一样？那个录像店更是，还有好多奇装异服的人出入，有人了解吗？
![1111111](https://github.com/user-attachments/assets/11fa588c-9350-49f2-a87d-2b961535a9f9)

**Comment by Lain0100** [comment]
上次我去那家录像店看到一个穿执事服的狼希人！

**Comment by 75020-maker** [comment]
录像店里有两麦片兄妹，生意还不错，我挺喜欢去买片的说~~

**Comment by RAVEN-L** [comment]
我在那里看到过一位会漂浮的女仆小姐！但是她带给店主的“手作便当”的味道……算了还是不要再提了，我至今没能忘掉那股鱼腥草加臭豆腐的味道@.@

**Comment by wushanef** [comment]
据说，传奇绳匠法厄同就住在那，是我一位自称法厄同大人的狗的朋友说的

**Comment by ShenYouJiaA** [comment]
放心了，治安局的朱鸢小姐都经常去录像店，都是良民

**Comment by zhongze20** [comment]
刚入行的新手吧，那个玩具店卖的都是音擎，听说老板挺厉害的，黑白通吃呢，那店开了这么久一点事都没有。不过价格相当实惠，普通人攒攒丁尼也可以买的起，我真心安利一波。不过收银台上的小姑娘是个圈外人，买音擎的时候就说买玩具就行了，可别把人家小姑娘吓走了

**Comment by SWYH134649** [comment]
去过好多次了，顺带一提买够5部录像带可以摸伊埃斯呢qwq
